.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,95 @@
+name: Bug Report
+description: Create a bug report to help us improve
+title: "[Bug]: "
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to fill out this bug report!
+  - type: markdown
+    attributes:
+      value: Please make sure to provide the information we ask for. This will allow us to help you better.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: A bug happened! 'foo' should not be there.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: A clear and concise description of what you expected to happen.
+      placeholder: I expected to see X or Y instead of 'foo'
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to reproduce
+      description: Please add link to a repo with a minimal reproduction. Or describe as detailed as possible how we can reproduce/verify the bug.
+      placeholder: |
+        Example steps (replace with your own):
+        1. Clone my repo at https://github.com/<myuser>/example
+        2. npm run minify
+        3. You should see the error come up
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: The version of cssnano you are using.
+      placeholder: ex. v5.0.6
```

I think no `v` avoids pontential confusion as there's no `v` in the version they're going to see in their `package.json`  (maybe we should change the **Releases** page on Github too)

```suggestion
      placeholder: ex. 5.0.6
```